### PR TITLE
Update repaying-your-loan.md

### PR DESCRIPTION
**Pull request by JamelleWT**
@KishaT language is pasted.

### CODE REVIEW

File: pages/loan-basics/repaying-your-loan.md [hunk 1]
```diff
@@ -12,3 +12,41 @@ scripts:
 permalink: /loan-basics/repaying-your-loan/
 ---
 
+# Repaying your loan
+ 
+## Loan payments
+#### Payroll deductions
```
Comment (line47) [inline pages/loan-basics/repaying-your-loan.md]: This should be a level 3 header `###`

File: pages/loan-basics/repaying-your-loan.md [hunk 2]
```diff
@@ -12,3 +12,41 @@ scripts:
...
+Check your leave and earnings statement to be sure that loan payments have started and that they are in the correct amount. Contact your agency or service if payments have not started or if they’re in the wrong amount.
+ 
+Be aware that you’re responsible for the repayment of your loan regardless of whether your agency or service misses a payment.
+<div class="usa-alert usa-alert-info">
+<div class="usa-alert-body" markdown="1">
+If you have two TSP accounts and you want to combine your accounts, you must close any loan in the account you are moving before the accounts can be combined.
+{: .usa-alert-text}
+</div>
+</div>
+#### Loan interest
```
Comment (line47) [inline pages/loan-basics/repaying-your-loan.md]: This should be a level 3 header `###`

File: pages/loan-basics/repaying-your-loan.md [hunk 3]
```diff
@@ -12,3 +12,41 @@ scripts:
...
+When you repay your loan, you repay it with interest. The repayment amount gets deposited back into your account and is invested according to your most recent [contribution allocation](#).
+ 
+Daily interest on your loan is calculated as each payment is posted and is based on the number of days since the last loan payment and the outstanding loan balance.
+Your loan interest payments are not tax deductible.
+
+#### Extra payments
```
Comment (line47) [inline pages/loan-basics/repaying-your-loan.md]: This should be a level 3 header `###`

File: pages/loan-basics/repaying-your-loan.md [hunk 4]
```diff
@@ -12,3 +12,41 @@ scripts:
...
+You can make extra loan payments (in addition to your payroll deduction) at any time using a personal check, cashier's check, or money order. You must send [Form TSP-26](#), _Loan Payment Coupon_, along with your extra payments.
+ 
+If you use an online banking service to make extra loan payments, be sure to refer to the Loan Payment Coupon to ensure that the information it requires gets included on your bank check. If any information is missing, the check will be returned.
+#### Missed payments
```
Comment (line47) [inline pages/loan-basics/repaying-your-loan.md]: This should be a level 3 header `###`

File: pages/loan-basics/repaying-your-loan.md [hunk 5]
```diff
@@ -12,3 +12,41 @@ scripts:
...
+You are responsible for ensuring that the loan payments are correct and submitted on time regardless of whether your agency or service missed your loan payment.
+ 
+#### Making up missed payments
```
Comment (line47) [inline pages/loan-basics/repaying-your-loan.md]: This should be a level 3 header `###`

File: pages/loan-basics/repaying-your-loan.md [hunk 6]
```diff
@@ -12,3 +12,41 @@ scripts:
...
+You can make extra loan payments (in addition to your payroll deduction) at any time using a personal check, cashier's check, or money order. You must send [Form TSP-26](#), _Loan Payment Coupon_, along with your extra payments.
+ 
+If you use an online banking service to make extra loan payments, be sure to refer to the Loan Payment Coupon to ensure that the information it requires gets included on your bank check. If any information is missing, the check will be returned.
+#### Missed payments
+You are responsible for ensuring that the loan payments are correct and submitted on time regardless of whether your agency or service missed your loan payment.
+ 
+#### Making up missed payments 
+At the end of each calendar quarter, we identify all loans with missed payments. If you have missed more than 2½ payments, we will send a notice letting you know that you have until the end of the following calendar quarter to pay the missed amount. You must pay the missed amount directly to us using your own personal funds to avoid a taxable distribution. Your payroll office cannot make up missed payments from your pay.
+ 
+#### Exceptions for missed payments
```
Comment (line47) [inline pages/loan-basics/repaying-your-loan.md]: This should be a level 3 header `###`

**Comment by KishaT** [comment]
@line47 and @JamelleWT I changed the headers from level 4 to 3 on this page. I will approve now.

**Pull request merged**